### PR TITLE
Boolean and YesNoBoolean support

### PR DESCRIPTION
**Pull request by yuvalherziger**
Tests included

### CODE REVIEW

**Comment by yuvalherziger** [comment]
Tests fail - I'm on it [UPDATE: fixed]

File: src/BooleanParser.php [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace MPScholten\RequestParser;
+
+class BooleanParser extends AbstractValueParser
+{
+    protected function parse($value)
+    {
+        if (!isset($value)) {
```
Comment (mpscholten) [inline src/BooleanParser.php]: Isn't `!isset($value)` always true? `parse` should never be called with `$value == null` (see https://github.com/mpscholten/request-parser/blob/master/src/AbstractValueParser.php#L17)

File: src/YesNoBooleanParser.php [hunk 2]
```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace MPScholten\RequestParser;
+
+class YesNoBooleanParser extends AbstractValueParser
+{
+    protected function parse($value)
+    {
```
Comment (mpscholten) [inline src/YesNoBooleanParser.php:8]: Same as above

File: src/BooleanParser.php (hunk 1 above)
Comment (yuvalherziger) [inline src/BooleanParser.php]: That's right, will change, thanks :)

File: src/YesNoBooleanParser.php (hunk 2 above)
Comment (yuvalherziger) [inline src/YesNoBooleanParser.php:8]: 👍

**Comment by mpscholten** [comment]
Good job :shipit:

**Pull request merged**